### DISCUSSION TITLE
Duda función qsort de la librería stdlib.h

### DISCUSSION THREAD

**Discussion by DvskyfenixX** [IIC2133-PUC/2024-2#144] [Tarea 2] [answered]
Holaa, estuve investigando un poco con respecto a algunas funciones generales de C y me encontré con la función de nombre "qsort" (proveniente de la librería de stdlib.h) que permite ordenar listas utilizando el algoritmo de quicksort pero sin la necesidad de tener que implementar este algoritmo como tal. Es por esto que me surgió la duda de si es posible utilizar esta función sin problemas o si se deben en realidad implementar directamente los algoritmos de búsqueda utilizados (en caso de utilizar alguno). 

Saludoss

**Comment by sebacarrascop** [comment, marked as answer]
hola!, si se puede usar, pero ten en cuenta que no es estable. Saludos.

**Reply by Mikson16** [reply]
En que sentido no es estable?

**Reply by sebacarrascop** [reply]
hola!, tengo entendido que eso se ve en clases.
Te adjunto la definición de wikipedia.
<img width="1368" alt="image" src="https://github.com/user-attachments/assets/75d5ac59-98f0-48c9-9094-1104d4a2014a">
